.github/workflows/2025-4.yml

```diff
@@ -0,0 +1,71 @@
+name: 2025-4 Nightly Build
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
github recommends avoiding scheduling workflows for minute 0 to avoid demand peaks

https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#schedule

Reply:
Done.